### PR TITLE
feat-grid

### PR DESCRIPTION
**Pull request by abhij1607**
image

### CODE REVIEW

**Review by stutisk — COMMENTED** [review]
this looks good to me,nice work abhi ,can go forward with merging it!

File: docs/css/utility.css [hunk 1]
```diff
@@ -45,3 +45,9 @@
 .box-shadow-dk {
   box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 15%);
 }
+.gap-1 {
+  gap: 1rem;
```
Comment (shivampandey0) [inline docs/css/utility.css:49]: variables can be used for units

**Review by shivampandey0 — COMMENTED** [review]
Looks good.
Added a suggestion

File: docs/css/utility.css (hunk 1 above)
Comment (abhij1607) [reply docs/css/utility.css:49]: thanks for the suggestion, I'll try to implement it later on😊

**Pull request merged**